deploy/fio.yaml

```diff
@@ -34,6 +34,8 @@ spec:
           value: "/volume/test"
         - name: SIZE
           value: "30G" # must be 10% smaller than the PVC size due to filesystem also took space
+        - name: CPU_IDLE_PROF
```

Comment:
Thanks for identifying the overhead! I don't think we should keep the option then. We need to think about another way to get the user aware of the potential CPU impact on the latency though.

Reply:
Agree. We need have a way to get average CPU usage by parallel.